src/EnergyPlus/DirectAirManager.cc

```diff
@@ -472,12 +493,43 @@ namespace DirectAirManager {
 
 		// Set the ZoneNode number
 		ZoneNode = DirectAir( DirectAirNum ).ZoneSupplyAirNode;
+
+		//TRANE BEG: DCV
+		Real64 mDotFromOARequirement( 0.0 );
+		if ( ! DirectAir( DirectAirNum ).NoOAFlowInputFromUser ) {
+			mDotFromOARequirement = DirectAir( DirectAirNum ).AirMassFlowRateMax;
+			int airLoopNum( 0 );
+			Real64 airLoopOAFrac( 0.0 );
+			if ( DirectAir( DirectAirNum ).ZoneEqNum > 0) {
+				airLoopNum =ZoneEquipConfig( DirectAir( DirectAirNum ).ZoneEqNum ).AirLoopNum;
+			}
+			if ( airLoopNum > 0 ) {
+				airLoopOAFrac = DataAirLoop::AirLoopFlow( airLoopNum ).OAFrac;
+				if ( airLoopOAFrac > 0.0 ) {
+					Real64 vDotOAReq = DataZoneEquipment::CalcDesignSpecificationOutdoorAir( DirectAir( DirectAirNum ).OARequirementsPtr, DirectAir( DirectAirNum ).ZoneNum, true, true );
+					Real64 rhoAir = PsyRhoAirFnPbTdbW( Node( DirectAir( DirectAirNum ).ZoneSupplyAirNode ).Press, Node( DirectAir( DirectAirNum ).ZoneSupplyAirNode ).Temp, Node( DirectAir( DirectAirNum ).ZoneSupplyAirNode ).HumRat );
+					mDotFromOARequirement = vDotOAReq * rhoAir / airLoopOAFrac;
```

Comment:
@mjwitte Agree. The Controller:MechanicalVentilation uses StdRhoAIr to get mass flow rate. It will be consistent everywhere. @EnergyArchmage What do you think?